src/Management/src/Endpoint/Actuators/ThreadDump/ThreadDumpV2Converter.cs

```diff
@@ -20,14 +20,14 @@ public override void Write(Utf8JsonWriter writer, IList<ThreadInfo> value, JsonS
 
         writer.WriteStartObject();
         writer.WritePropertyName("threads");
-        writer.WriteStartObject();
+        writer.WriteStartArray();
```

Comment:
This change definitely seems correct, but how did you discover the error?

Reply:
During the implementation of `ActuatorsHostBuilderTest`, I'd typically write the test, then put a breakpoint on the returned response body text, and then paste it in a tool to reformat the returned JSON, to determine what to assert on. The tool reported the pasted JSON was invalid.